src/crimson/osd/osd_operation_sequencer.h

```diff
@@ -31,11 +31,11 @@ class OpSequencer {
             typename Result = std::invoke_result_t<FuncT>>
   seastar::futurize_t<Result>
   start_op(HandleT& handle,
-           uint64_t prev_op,
-           uint64_t this_op,
+           const Operation* const prev_op,
```

Comment:
might want to expand a little bit on the rationales behind this change in the commit message.

Reply:
Sure.